src/components/DisplayBox.js

```diff
@@ -246,7 +246,10 @@ retrieveLaunchContext(link, accessToken, patientId, fhirBaseUrl) {
               const card = JSON.parse(JSON.stringify(c));
       
               // -- Summary --
-              const summarySection = <Text fontSize={18} weight={700} color={summaryColors[card.indicator]}>{card.summary}</Text>;
+              const summarySection = <Text fontSize={18} weight={700} color={summaryColors[card.indicator]}>
```

Comment:
The detail should be put into the actual detail section unless you have a reason otherwise.  See a couple lines below, the detail section is just an empty string.  It should be 

`        const detailSection = card.detail ? <ReactMarkdown escapeHtml={false} softBreak="br" source={card.detail} /> : '';`